web/src/components/repo/build/BuildItem.vue

```diff
@@ -19,7 +19,8 @@
 
     <div class="flex py-2 px-4 flex-grow min-w-0 <md:flex-wrap">
       <div class="<md:hidden flex items-center flex-shrink-0">
-        <img class="w-8" :src="build.author_avatar" />
+        <Icon v-if="build.event === 'cron'" name="stopwatch" class="text-color" />
```

Comment:
Should we maybe use the woodpecker icon as avatar to show its executed by the system / cron and instead show the stopwatch as event icon as the `push` icon is somehow not 100% suited there? 🤔

Reply:
I like it the way it is atm ... but we could do so too ... we just have to indicate it's a cron somehow else then

Reply:
I tired the other way, but wasn't really happy with the result so we can stick to this for now IMO.